src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -1671,86 +1671,86 @@
+        "RESUME_ALL_SUCCESS": "Reprise des tâches réussie",
+        "VENDOR_TYPE": "Type de fournisseur",
+        "VENDOR_ID": "ID du fournisseur",
+        "NO_SCHEDULE": "Nous n'avons trouvé aucune programmation",
         "WORKERS": "Workers",
-        "FREE_ALL": "Free all",
-        "CONFIRM_FREE_ALL": "Confirm Freeing All",
-        "CONFIRM_FREE_ALL_CONTENT": "Do you want to free all the workers?",
-        "CONFIRM_FREE_WORKERS": "Confirm Freeing Workers",
-        "CONFIRM_FREE_WORKERS_CONTENT": "Do you want to free the workers {{param}}?",
-        "FREE_WORKER_SUCCESS": "Freed workers successfully",
-        "FREE_ALL_SUCCESS": "Freed all the workers successfully",
-        "WORKER_POOL": "Worker Pools",
-        "WORKER_POOL_ID": "Worker Pool ID",
+        "FREE_ALL": "Tout libérer",
+        "CONFIRM_FREE_ALL": "Confirmer la libération de tous les workers",
+        "CONFIRM_FREE_ALL_CONTENT": "Voulez-vous libérer tous les workers ?",
+        "CONFIRM_FREE_WORKERS": "Confirmer la libération de tous les workers",
+        "CONFIRM_FREE_WORKERS_CONTENT": "Voulez-vous libérer les workers {{param}} ?",
+        "FREE_WORKER_SUCCESS": "Libération des workers réussie",
+        "FREE_ALL_SUCCESS": "Libération de tous les workers réussie",
+        "WORKER_POOL": "Pools de workers",
+        "WORKER_POOL_ID": "ID du pool de worker",
         "PID": "Pid",
-        "START_AT": "Started At",
-        "HEARTBEAT_AT": "Heartbeat At",
-        "CONCURRENCY": "Concurrency",
-        "NO_WORKER_POOL": "We could not find any worker pool",
-        "FREE": "Free",
-        "WORKER_ID": "Worker ID",
-        "JOB_ID": "Job ID",
-        "CHECK_IN_AT": "Checked In At",
-        "NO_WORKER": "We could not find any worker",
-        "JOB_QUEUE": "Job Queues",
-        "JOB_SERVICE_DASHBOARD": "Job Service Dashboard",
-        "OPERATION_STOP_ALL_QUEUES": "Stop all job queues",
-        "OPERATION_STOP_SPECIFIED_QUEUES": "Stop specified job queues",
-        "OPERATION_PAUSE_SPECIFIED_QUEUES": "Pause specified job queues",
-        "OPERATION_RESUME_SPECIFIED_QUEUES": "Resume specified job queues",
-        "OPERATION_PAUSE_SCHEDULE": "Pause all schedules",
-        "OPERATION_RESUME_SCHEDULE": "Resume all schedules",
-        "OPERATION_FREE_ALL": "Free all workers",
-        "OPERATION_FREE_SPECIFIED_WORKERS": "Free specified workers",
-        "QUEUE_STOP_BTN_INFO": "STOP — Stop and remove all jobs in the selected queue(s).",
-        "QUEUE_PAUSE_BTN_INFO": "PAUSE — Pause the execution of jobs in this type of job queue. Jobs can be enqueued when the queue is paused.",
-        "QUEUE_RESUME_BTN_INFO": "RESUME — Resume the execution of jobs in this type of job queue.",
-        "SCHEDULE_PAUSE_BTN_INFO": "PAUSE — Pause all schedules to execute.",
-        "SCHEDULE_RESUME_BTN_INFO": "RESUME — Resume all schedule to execute.",
-        "WORKER_FREE_BTN_INFO": "Stop the current running job to free the worker",
+        "START_AT": "Démarré à",
```

Comment:
If english text is "started", than in FR it's "Démarré".
If it's "start", in FR it's "Démarrer"

Reply:
In english translation file it's `Started At`